aws-android-sdk-kinesisvideo/src/main/java/com/amazonaws/kinesisvideo/encoding/ChunkDecoder.java

```diff
@@ -53,7 +53,7 @@ private ChunkDecoder() {
 
         try {
             final String headersAsString =
-                    readInputStream(inputStream, PAYLOAD_DELIMITER.getBytes(StandardCharsets.UTF_8));
```

Comment:
`StandardCharsets` bumps the `minSdkVersion` up to 19, for no good reason. Don't use it.